requirements.txt

```diff
@@ -2,7 +2,7 @@
 matplotlib>=3.0.3
 numpy>=1.17.2
 pandas>=0.25.1
-scikit-learn==0.21.3
+scikit-learn>=0.21.3
 scipy>=1.3.1
```

Comment:
also requirements-fixed.txt? (I think that setup.py is fine though.)

Reply:
The point of `requirements-fixed.txt` is that versions are pinned.